### PR TITLE
WRP-7017: Update dependencies

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to the latest version.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated dependencies the below to the latest major version.
- "lerna": "5.6.1" -> "^6.6.2"
(no breaking change: https://dev.to/nx/upgrade-your-lerna-workspace-make-it-fast-and-modern-3c0g)

Updated other dependencies to the latest minor version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Fix travis fails

### Links
[//]: # (Related issues, references)
WRP-7017

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,11 +18,11 @@ install:
     - npm run bootstrap
 script:
     - echo -e "\x1b\x5b35;1m*** Starting tests...\x1b\x5b0m"
-    - npm test -- -- -- --runInBand --coverage
+    - npm test -- -- --runInBand --coverage
```
Comment (MikyungKim) [inline .travis.yml:21]: Would you be more specific for this change?

Comment (hong6316) [reply .travis.yml:21]: I cannot found about `--` argument on lerna docs(https://lerna.js.org/docs/introduction)
But these changes can fix the travis.

**Comment by hong6316** [comment]
Travis fail occurred on `nvm install` failed.
> Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
nvm: install v19.9.0 failed!
Failed to install 19. Remote repository may not be reachable.
`nvm install` failed

This issue is a travis environment issue and PR has already passed the travis test.
Do we ignore this travis fail? or lets not use node 19?

**Comment by MikyungKim** [comment]
We can't merge this as-is. Please re-run and if it occurs again, fix the travis problem.

**Comment by MikyungKim** [comment]
Let's keep the lockfile version as-is.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**